src/net/http.rs

```diff
@@ -1898,6 +1911,10 @@ impl HttpResponseType {
                 HttpResponsePreamble::new_serialized(fd, 200, "OK", md.content_length.clone(), &HttpContentType::Text, md.request_id, |ref mut fd| keep_alive_headers(fd, md))?;
                 HttpResponseType::send_text(protocol, md, fd, &txid_bytes)?;
             },
+            HttpResponseType::OptionsPreflight(ref md) => {
+                HttpResponsePreamble::new_serialized(fd, 200, "OK", None, &HttpContentType::Bytes, md.request_id, |ref mut fd| keep_alive_headers(fd, md))?;
```

Comment:
Content type should be text

Reply:
Also, you should replace the call to `keep_alive_headers()` with a function that wraps `keep_alive_headers()` and adds the CORS headers as well.

Reply:
Should be good now